praise-bot/functions/save_praise.ts

```diff
@@ -0,0 +1,105 @@
+import { DefineFunction, Schema, SlackFunction } from "deno-slack-sdk/mod.ts";
+import { SlackAPIClient } from "deno-slack-sdk/types.ts";
+import { isDebugMode } from "./internals/debug_mode.ts";
+
+/**
+ * Defines the SavePraiseFunction input and output parameters
+ */
+export const SavePraiseFunction = DefineFunction({
+    callback_id: "save_praise",
+    title: "Save a Praise",
+    description: "Saves data about a praise based on user input choices",
+    source_file: "functions/save_praise.ts", // The file with the exported function handler
+    input_parameters: {
+      properties: {
+        sender: {
+          type: Schema.slack.types.user_id,
+          description: "The user id of the person sending the praise",
+        },
+        doer: {
+          type: Schema.slack.types.user_id,
+          description: "The user id of the person receiving the praise",
+        },
+        channel: {
+            type: Schema.slack.types.channel_id,
+            description: "The channel where the praise is to be sent",
+        },
+        message: {
+            type: Schema.types.string,
+            description: "The message content of the praise",
+        },
+        vibe: {
+            type: Schema.types.string,
+            description: "The energy of the praise",
+        },                
+      },
+      required: ["doer", "channel", "message"],
```

Comment:
Is "sender" not also required here @AndrewHUNGNguyen ?

Reply:
I was debating whether or not to make it as a required variable.